python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -38,6 +41,223 @@
 __all__ = ["OnDiskDataset", "preprocess_ondisk_dataset", "BuiltinDataset"]
 
 
+def _graph_data_to_fused_csc_sampling_graph(
+    dataset_dir: str,
+    graph_data: Dict,
+    include_original_edge_id: bool,
+) -> FusedCSCSamplingGraph:
+    """Convert the raw graph data into FusedCSCSamplingGraph.
+
+    Parameters
+    ----------
+    dataset_dir : str
+        The path to the dataset directory.
+    graph_data : Dict
+        The raw data read from yaml file.
+    include_original_edge_id : bool
+        Whether to include the original edge id in the FusedCSCSamplingGraph.
+
+    Returns
+    -------
+    sampling_graph : FusedCSCSamplingGraph
+        The FusedCSCSamplingGraph constructed from the raw data.
+    """
+    is_homogeneous = (
+        len(graph_data["nodes"]) == 1
+        and len(graph_data["edges"]) == 1
+        and "type" not in graph_data["nodes"][0]
+        and "type" not in graph_data["edges"][0]
+    )
+
```

Comment:
`import dgl.sparse as dglsp`
Would placing the dgl.sparse import here instead of the top of the file help solve the issue?

Reply:
I think `dgl.sparse` and `dgl.graphbolt` should not depend on each other. We should also make an effort to reduce the dependency of graphbolt to even dgl.

Reply:
I see. Thanks for you kind advice!

Reply:
Looks like the cugraph test passed. #7077 and #7075 should not be needed anymore I guess.

Reply:
Yes. thank you again